edwardpizarro-fatima.js

```diff
@@ -0,0 +1,222 @@
+  // For example, the transmogrified result of 5, 3, and 2 is (5 times 3) to the power of 2 is 225.
+  // console.log(transmogrify(5, 3, 2));
+  
+  const transmogrify = (num1,num2,num3) => {
+      let newNum = num1 * num2;
+      let newNum2 = Math.pow(newNum,num3)
+  return newNum2;
+  }
+  
+  transmogrify(5,3,2)
+  //Trez broke this down and explained to me what was going on after Brunos office hours. I Have a decent understanding.
+  
+  //transmogrify = (num1, num2, num3) => {
+  //   return Math.pow(num1 * num2 * num3)
+  // }
+  // console.log(transmogrify(5, 3, 2))
+  //Mateen's solution
+  
+  
+  
+  //Question 7
+  //Write a function that takes a parameter, a number. The function should print the Fibonacci sequence up to that number.
+  
+  // Adjust the function to push the even numbered values into an array.
+  
+  // Adjust the function to return the summed value of the array.
+  
+  // Find the sum of the even numbered values that do not exceed four million.
+  // WRITE COMMENT
+  const fibonacci = (num) => {
+        const fibArray = [0,1]
+        for(let i = 2; i <= num; i++) {
+            const prevNum1 = fibArray[i-1]
+            const prevNum2 = fibArray[i-2]
+            fibArray.push(prevNum1 + prevNum2)
+  }
+      return fibArray[num]
+  }
+  fibonacci(20)
+  //Bruno went over this with some other students in office hours and would love extra help on this. I would say Im not super comfortable with whats going on. I have the code down to break it down and really analyze whats going on.
+  
+  
+  
+  
+  //const fib = (limit) => {
+  // let arr = [0]
+  // let evenArr = []
+  
+  // for(let i = 0; i < 100; i++){
+  //   if(arr[arr.length - 1] <= limit){ 
+  //     if(i > 1){
+  //       arr.push(arr[i -1] + arr[i - 2])
+  //     } else {
+  //       arr.push(i)
+  //     }
+  //   }
+  // }
+  // for(index of arr){
+  // if(index % 2 === 0)
+  //   }
+  // let sum = arr.reduce((acc, current) => {
+  //   return acc + current
+    
+  // }
+  
+  //   return arr
+  
+  // }
+  
+  // console.log(fib(40))
+  //METEENS SOLUTION ^
+  
+  
+  
+  
+  
+  
+  //Can you find the needle in the haystack?
+  
+  // Write a function findNeedle() that takes an array full of junk but contains one "needle"
+  
+  // After your function finds the needle it should return a message (as a string) that says:
+  
+  // "found the needle at postition" plus the index it found the needle so:
+  
+  // findNeedle(['hay', 'junk', 'hay', 'hay', 'moreJunk', 'needle', 'randomJunk'])
+  
+  // Should return:
+  
+  // "found the needle at position 5"
+  
+  // let findNeedle = ['grass', 'tree', 'needle', 'junk', 'house'];
+  
+  // findNeedle(findNeedle);
+  
+  const findNeedle = (haystack) => {
+    return "found the needle at position " + haystack.indexOf("needle");
+  }
+  
+  findNeedle(['car', 'junk', 'hay', 'needle', 'house', 'junk', 'random'])
```

Comment:
nice! `indexOf()` is definitely the easiest way of getting the answer. would you be able to figure out a solution if you didn't have that method, though?